openghg/retrieve/_search.py

```diff
@@ -491,7 +492,9 @@ def _base_search(**kwargs: Any) -> SearchResults:
     # - clean search terms directly or within data structures
     search_kwargs = {}
     for k, v in kwargs.items():
-        if isinstance(v, (list, tuple)):
+        if k.lower() in ["inlet", "height"]:
```

Comment:
Based on the [metadata_default_keys function](https://github.com/openghg/openghg/blob/devel/openghg/standardise/meta/_metadata.py#L12) the other keys this would impact would be:
 - "inlet_height_magl"
 - "station_height_masl"

When we tidy up formatting, it would be good to have a record of which key names we expect this formatting to be needed for.